paywall/src/data-iframe/blockchainHandler/keyStatus.js

```diff
@@ -42,8 +42,7 @@ export function getKeyStatus(key, requiredConfirmations) {
 /**
  * Construct the transactions field for each key
  * @param {object} keys keys, indexed by their lock/owner ID (not the same as smart contract ID)
- * @param {object} transactions transactions, indexed by hash (submitted transaction is always
```

Comment:
correct a misunderstanding. This is not directly related to this PR, but is indirectly related